README.md

```diff
@@ -1,7 +1,61 @@
 # SofaGym
 
-Software toolkit to easily create an OpenAI Gym environment out of any SOFA scene.
-The toolkit provides an API based on the standard OpenAI Gym API, allowing to train classical Reinforcement Learning algorithms. The toolkit also comprises example scenes based on the SoftRobots plugin for SOFA to illustrate how to include SOFA simulations and train learning algorithms on them.
+Software toolkit to easily create an [OpenAI Gym](https://github.com/openai/gym) environment out of any [SOFA](https://github.com/sofa-framework/sofa) scene.
+
+The toolkit provides an API based on the standard OpenAI Gym API, allowing to train classical Reinforcement Learning algorithms. 
+
+The toolkit also comprises example scenes based on the SoftRobots plugin for SOFA to illustrate how to include SOFA simulations and train learning algorithms on them.
+
+## Installation
+### Prerequisites
+
+#### 1) SofaGym need a [Sofa installation](https://www.sofa-framework.org/community/doc/getting-started/build/linux/) v22.06
+with some mandatory plugins :
+* [SofaPython3](https://github.com/sofa-framework/SofaPython3) (fetchable within sofa)
+* [BeamAdapter](https://github.com/sofa-framework/BeamAdapter) (fetchable within sofa)
+* [SPLIB](https://github.com/SofaDefrost/SPLIB)
+* [STLIB](https://github.com/SofaDefrost/STLIB)
+* [SoftRobots](https://github.com/SofaDefrost/SoftRobots)
```

Comment:
Why BeamAdapter and SoftRobots are not optional?

Reply:
They are used in almost all of the envs examples. However, actually, they are not really mandatory. I will check that and correct accordingly.